### ISSUE TITLE
The UK is a monarchy

### ISSUE THREAD

**Issue by MostAwesomeDude** [nixos-users-for-western-mil-and-govs/NixOS-Users-For-Western-MIL-And-GOVs.github.io#6]
Right now there is a boast of "democratic governance" in the letter. However, as explained in #3, "Western" can refer to NATO members or their neighbors and allies, and as explained in #5, the UK is included. This is a contradiction, because the UK is a monarchy which does not espouse democratic values, refusing to let its occupied territories of Scotland and Northern Ireland determine their own governance and forcefully withdrawing them from the European Union.

Possible resolutions include:

* Remove "democratic governance" from the letter
* Explicitly disclaim the UK as non-representative of "democratic governance"

**Comment by maxkokocom** [comment]
Right, I will remove the UK, and I will include North Korea as, technically, they are a "democratic republic". 

It's all jokes to you guys. Meanwhile, you are removing any chances for NixOS to go mainstream. From now on, no company should ever consider NixOS with the risk of the community changing a license to a non-open source at random, and no engineer should invest time in Nixos's skillset now. You are creating a marvel just to kill it.

**Comment by MostAwesomeDude** [comment]
You sound unfamiliar with the concept of Free Software. We can survive without corporate support. Try running Gentoo sometime -- a binary cache is a luxury, not a necessity.

**Comment by maxkokocom** [comment]
Yes. I am unfamiliar with the Free Software concept or the implications of different licenses. If I were familiar with it, I would obviously agree with you!

Gentoo is a great Linux distro success example! It's excellent that NixOS chose it as its role model.

**Comment by maxkokocom** [comment]
PS I love reacting with emojis to your own posts. Should learn this concept from you as well.

**Comment by drakon64** [comment]
> the UK is a monarchy which does not espouse democratic values

Out of all of the reasons you could have cited that the UK is not a democracy, you chose the one that isn't remotely correct. The monarchy has no real political power, they're effectively ceremonial.

> refusing to let its occupied territories of Scotland and Northern Ireland determine their own governance

[Both](https://en.wikipedia.org/wiki/Scottish_Parliament) [territories](https://en.wikipedia.org/wiki/Northern_Ireland_Assembly) have their own devolved parliaments. [Scotland also held an independence referendum in 2014 in which they rejected independence.](https://en.wikipedia.org/wiki/2014_Scottish_independence_referendum)

What happened to the time where we could peacefully discuss our differences instead of resorting to fabrications?

**Comment by MostAwesomeDude** [comment]
Hi @drakon64 , thanks for contributing! Please consider signing the letter if you believe that it is representative of your values. You could be the person who takes this letter from "user" to "users"; see #7 for more details.

**Comment by drakon64** [comment]
> Hi @drakon64 , thanks for contributing! Please consider signing the letter if you believe that it is representative of your values. You could be the person who takes this letter from "user" to "users"; see #7 for more details.

@MostAwesomeDude I don't intend to sign any of the open letters, I merely take issue with your bad faith issues being raised.

**Comment by MostAwesomeDude** [comment]
@drakon64 Ah! I see. This repository and accompanying letter is a bad-faith reply (a ["shitpost"](https://en.wikipedia.org/wiki/Shitposting)) to [this open letter](https://nixos-users-against-mic-sponsorship.github.io/) (not the recent one); the grumble you have is with this repository's author — a non-contributor to nixpkgs — not with me.

**Comment by drakon64** [comment]
> @drakon64 Ah! I see. This repository and accompanying letter is a bad-faith reply (a ["shitpost"](https://en.wikipedia.org/wiki/Shitposting)) to [this open letter](https://nixos-users-against-mic-sponsorship.github.io/) (not the recent one); the grumble you have is with this repository's author — a non-contributor to nixpkgs — not with me.

@MostAwesomeDude No, I have a problem with both. Have you considered simply not engaging with a 'shitpost' rather than contributing to the current dumpster fire? There's already been controversy due to a prominent member of the Nix community (rightly or wrongly) being banned for wilfully furthering the division within the community.

It's also not a good look to attempt to discredit the author based on them being a 'non-contributor'. How is this relevant when the original open letter is from NixOS users and not Nixpkgs contributors?

**Comment by MostAwesomeDude** [comment]
@drakon64 It's been a week. I'm sure you've heard about Aux and Lix. Given that the repository author can't even bother to merge a second signature, are you *sure* that you want to defend this particular approach? It doesn't sound like a winning move.
